save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/utils/DateTimeUtils.kt

```diff
@@ -27,6 +28,15 @@ fun LocalDateTime.prettyPrint(timeZone: TimeZone = TimeZone.UTC) = toInstant(Tim
     .replace("Z", "")
     .replace("-", ".")
 
+/**
+ * @param timeZone timezone to print the date time in
+ * @return very pretty string representation of [LocalDateTime]
+ */
+fun LocalDateTime.veryPrettyPrint(timeZone: TimeZone = TimeZone.UTC) = toInstant(TimeZone.UTC).toLocalDateTime(timeZone)
```

Comment:
Supper very pretty print is coming 😂
Probably need to rename those methods according to format names

Reply:
Moreover, suggestion to make seconds, minutes etc have exactly 2 characters (04 instead of 4)

Reply:
Removed seconds

Reply:
Fixed